gcovr/gcov_parser.py

```diff
@@ -959,58 +972,72 @@ def _find_excluded_ranges(
     *,
     warnings: _ExclusionRangeWarnings,
     exclude_lines_by_pattern: Optional[str] = None,
+    exclude_branches_by_pattern: Optional[str] = None,
     exclude_pattern_prefix: str,
 ) -> Callable[[int], bool]:
     """
-    Scan through all lines to find line ranges covered by exclusion markers.
+    Scan through all lines to find line ranges and branch ranges covered by exclusion markers.
 
     Example:
-    >>> lines = [(11, '//PREFIX_EXCL_LINE'), (13, '//IGNORE'), (15, '//PREFIX_EXCL_START'), (18, '//PREFIX_EXCL_STOP')]
-    >>> exclude = _find_excluded_ranges(
-    ...     lines, warnings=..., exclude_lines_by_pattern = '.*IGNORE', exclude_pattern_prefix='PREFIX')
-    >>> [lineno for lineno in range(20) if exclude(lineno)]
+    >>> lines = [(11, '//PREFIX_EXCL_LINE'), (13, '//IGNORE_LINE'), (15, '//PREFIX_EXCL_START'), (18, '//PREFIX_EXCL_STOP'),
+    ...     (21, '//PREFIX_EXCL_BR_LINE'), (23, '//IGNORE_BR'), (25, '//PREFIX_EXCL_BR_START'), (28, '//PREFIX_EXCL_BR_STOP')]
```

Comment:
Do we want to have something different than the default LCOV marker `LCOV_EXCL_BR_LINE` ?

Since the line exclusion markers are compatible, I would have expect the same for the branch ones. 

http://ltp.sourceforge.net/coverage/lcov/geninfo.1.php

Reply:
LCVO wasn't in my mind. if there `BR` is used please ignore the suggestions regarding the renaming.